pkg/utils/parse.go

```diff
@@ -9,6 +9,7 @@ import (
 
 var ParseErr = errors.New("parse-err")
 
+// Reads and unmarshals content from buffer.
 func Parse(read io.ReadCloser, target any) error {
```

Comment:
So if it parses json, maybe we will name it ParseJSON?

Reply:
Good point. Renamed